### ISSUE TITLE
Admin section

### ISSUE THREAD

**Issue by Linesmerrill** [Linesmerrill/police-cad#563]
**Describe the solution you'd like**
[_A clear and concise description of what you want to happen._]

- Admin System
    - So admins can choose who sees the police section
- Civ section
- EMS/Fire section

**Implementation Details**
[_Add any and all specific things you would like to see implemented. These can include things such as limiting the input for a input box to only include specific characters. (This can be as technical as you would like to make it, including code examples)_]

-

**Comment by SowinskiBraeden** [comment]
Maybe have a set of permissions linked to users.
The Admins end generates a list of users in their community, by default they have the civilian permission.
Then they can give or remove permissions such as police, dispatch, fire/ems, possibly even civilian like a suspension from their community.

I would also assume they would want the ability to kick members, possibly ban

Finally I think they'd like to give a admin permission to users to do the same as them.

**Comment by Linesmerrill** [comment]
Yup 💯 agree with you. I don't think it would be impossible to do, but would prob be a bit of work.

**Comment by Linesmerrill** [comment]
> Describe the solution you'd like
> 
> Civ Section (Authorized)
> Police Section (Unauthorized)
> Implementation Details
> 
> I want a system where you can pick what sections a user can go through, rather then having them open to all sections.
> This could be possibly done by having a different invite code for every section which the user joins through or another  system which involves the admins, like manual verification and authorization.

**Comment by Linesmerrill** [comment]
Done, download our mobile app and have full control over your community.
